spec/unit/helpers/check_helper_production_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'gooddata/lcm/actions/base_action'
+require 'gooddata/lcm/helpers/check_helper'
+require 'gooddata/lcm/types/types'
+
+describe GoodData::LCM2::Helpers::Check do
+  let(:params) do
+    params = { test_param_four: 4 }
+    GoodData::LCM2.convert_to_smart_hash(params)
+  end
+  context 'when running outside the tests' do
+    let(:mocked_logger) { double(Logger) }
+    before do
+      expect(GoodData).to receive(:logger).and_return(mocked_logger)
+      expect(ENV).to receive(:[]).with('RSPEC_ENV').and_return('production')
+      expect(mocked_logger).to receive(:error)
+    end
+    it 'verifies required and writes out error message but do not fail' do
+      PARAMS = GoodData::LCM2::BaseAction.define_params(self) do
+        description 'Testing param one'
+        param :test_param_one, instance_of(GoodData::LCM2::Type::StringType), required: true
+      end
+      expect { GoodData::LCM2::Helpers.check_params(PARAMS, params) }.not_to raise_error(/Mandatory/)
+    end
+
+    it 'checks types and writes out error message but do not fail' do
```

Comment:
Similar code found in check_helper_production_spec.rb:25, check_helper_spec.rb:40 (mass = 70)